specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/TasksCommon.json

```diff
@@ -326,6 +360,81 @@
         "targetConnectionInfo"
       ]
     },
+    "MigrateSqlServerSqlDbDatabaseInput": {
+      "type": "object",
+      "description": "Database specific information for SQL to Azure SQL DB migration task inputs",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the database"
+        },
+        "targetDatabaseName": {
+          "type": "string",
+          "description": "Name of target database. Note: Target database will be truncated before starting migration."
+        },
+        "makeSourceDbReadOnly": {
+          "type": "boolean",
+          "description": "Whether to set database read only before migration"
+        },
+        "tableMap": {
+          "type": "object",
+          "description": "Mapping of source to target tables",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "MigrateSqlServerSqlDbSyncDatabaseInput": {
+      "type": "object",
+      "description": "Database specific information for SQL to Azure SQL DB sync migration task inputs",
+      "properties": {
+        "targetDatabaseName": {
+          "type": "string",
+          "description": "Target database name"
+        },
+        "schemaName": {
+          "type": "string",
+          "description": "Schema name to be migrated"
+        },
+        "tableMap": {
+          "type": "object",
+          "description": "Mapping of source to target tables",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "migrationSetting": {
+          "type": "object",
+          "description": "Migration settings which tune the migration behavior",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "sourceSetting": {
+          "type": "object",
+          "description": "Source settings to tune source endpoint migration behavior",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "targetSetting": {
+          "type": "object",
+          "description": "Target settings to tune target endpoint migration behavior",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "id": {
```

Comment:
nit: Can we move id and name to the top?

Reply:
Done.